fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -18,7 +20,11 @@
      * @deprecated in favor of loadMetricDictionary(MetricDictionary, DimensionDictionary)
      */
     @Deprecated
-    void loadMetricDictionary(MetricDictionary metricDictionary);
+    default void loadMetricDictionary(MetricDictionary metricDictionary) {
+        String message = "The method loadMetricDictionary(MetricDictionary, DimensionDictionary) should be used " +
+                "instead";
+        LoggerFactory.getLogger(MetricLoader.class).warn(message);
+    }
```

Comment:
I would lift this logger up to a static final on the class just to keep with the pattern used everywhere else